servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/SenderErrorContext.java

```diff
@@ -7,6 +7,7 @@
 
 public class SenderErrorContext extends ErrorContext
 {
+    private static final long serialVersionUID = -8426189357575601244L;
```

Comment:
[SpotBugs]
Since ServiceBusException has serialVersionUID, it means it needs to implement Serializable.  Change the ErrorContext to implements Serializable cause all its sub-extended class to have the serialVersionUID